src/events/models/Attendee.ts

```diff
@@ -1,10 +1,8 @@
-import { IUser } from 'core/models/User';
 import { IAttendanceEvent } from './Event';
 import { IExtra } from './Extras';
 
 export interface IAttendee {
   event: IAttendanceEvent;
-  user: IUser;
```

Comment:
Does this mean that we no longer have an Online-user linked to an attendee? How isn't that going to affect our opportunity to implement the "Show attending"-feature? (For the events with public attendee-lists)

Reply:
Sort of a temporary fix this one. I can set up a correct model for it.

The problem with public attendee lists resides with an another issue entirely, which I will leave to the developer implementing the event registration.

This model can not be used to display those users, as it contains more information than what we want to be displayed through the API for attending users.